<launch> 
  <include file="$(find turtlebot_bringup)/launch/3dsensor.launch">
    <arg name="rgb_processing"                  value="true"/>  <!-- only required if we use android client -->
    <arg name="depth_processing"                value="true"/>
    <arg name="depth_registered_processing"     value="false"/>
    <arg name="depth_registration"              value="false"/>
    <arg name="disparity_processing"            value="false"/>
    <arg name="disparity_registered_processing" value="false"/>
    <arg name="scan_processing"                 value="false"/>
  </include>

<node pkg="hd_detector" type="getCloudview" name="CloudViewer">
<!-- <remap from="depth/points" to="camera/depth/points"/> -->
</node>


  <!-- Sandalone launching: useful for testing 
  <node pkg="nodelet" type="nodelet" name="turtlebot_follower_manager" args="manager"/>
  <node pkg="nodelet" type="nodelet" name="turtlebot_follower" args="load turtlebot_follower/TurtlebotFollower turtlebot_follower_manager"> -->
</launch>
